addon/components/horizontal-bar-chart.js

```diff
@@ -237,13 +237,24 @@ const HorizontalBarChartComponent = ChartComponent.extend(FloatingTooltipMixin,
     };
   }),
 
+  /*
+   * Ensures that grouping labels will be on the left when all data is negative or 0
+   * @private
+   * @param {Object}
+   * @return {Boolean}
+   */
+  _isNotPositive: function(d) {
```

Comment:
I'm not sure how I feel about this name. Technically anything <= 0 is not positive, this name doesn't really capture the `this.get('hasAllNegativeValues')` bit.